.github/workflows/test.yml

```diff
@@ -0,0 +1,29 @@
+name: Test
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
```suggestion
      - main
```

I used main for the default branch

Reply:
Good catch, thank you. Turns out it was an issue on all of the workflows so updated now.